python/ray/rllib/common.py

```diff
@@ -0,0 +1,33 @@
+from collections import namedtuple
+
+
+TrainingResult = namedtuple("TrainingResult", [
+    "training_iteration",
+    "episode_reward_mean",
+    "episode_len_mean",
+])
+
+
+class Algorithm(object):
+  """
+  All RLlib algorithms extend this base class.
+
+  Algorithm objects retain internal model state between calls to train(), so
+  you should create a new algorithm instance for each training session.
+
+  TODO(ekl) support checkpoint / restore of training state.
```

Comment:
Let's stick with `TODO(ekl):` for TODOs. Also let's stick with the google style docstrings described here http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.